lib/src/modules/navigationItems/NavigationItem.js

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import cx from 'classnames';
+import { NavigationContext } from './NavigationProvider';
+
+export function NavigationItem({ children, id, className }) {
+  const { setActiveTabId, activeTabId } = useContext(NavigationContext);
+  const isActive = activeTabId === id;
+
+  const classNames = cx(className, 'navigation--item', {
+    active: isActive
+  });
+  return (
+    <li className={classNames} role="none">
+      <button
+        onClick={() => setActiveTabId(id)}
```

Comment:
🚩 Do we also want to pass our own `onClick` prop so we can control what it does? 😅  (unless i'm mistaking how we'll use this!)

Reply:
Good shout. 

I guess we could digest the active tab (from context) and `useEffect` on that but maybe that isn't clear? I'm easy.